.github/workflows/_push-entrypoint.yaml

```diff
@@ -33,7 +33,7 @@ jobs:
       ELIXIR_VSN: ${{ steps.env.outputs.ELIXIR_VSN }}
       BUILDER: ${{ steps.env.outputs.BUILDER }}
       BUILD_FROM: ${{ steps.env.outputs.BUILD_FROM }}
-      RUN_FROM: ${{ steps.env.outputs.BUILD_FROM }}
+      RUN_FROM: ${{ steps.env.outputs.RUN_FROM }}
```

Comment:
@id  these two lines are missing in master branch, is it intended ?

Reply:
Yes.